apps/namehashlabs.org/components/2 - molecules/header.tsx

```diff
@@ -9,22 +10,22 @@ export const Header = () => {
   return (
     <nav className="lg:px-[112px] px-5 w-full h-20 flex items-center justify-center border-b border-black border-opacity-10 z-20">
```

Comment:
You're right. The `Link` component was designed to have no hover styles on mobile, because there isn't really hover state, but an active state.

I've reverted that change and applied the same style to all viewports now.